Sources/ComposableArchitecture/Scheduling/TestScheduler.swift

```diff
@@ -55,8 +55,8 @@ where SchedulerTimeType: Strideable, SchedulerTimeType.Stride: SchedulerTimeInte
   public func schedule(
     after date: SchedulerTimeType,
     interval: SchedulerTimeType.Stride,
-    tolerance: SchedulerTimeType.Stride,
-    options: SchedulerOptions?,
+    tolerance _: SchedulerTimeType.Stride,
+    options _: SchedulerOptions?,
```

Comment:
unrelated but snuck this in. while working on some episode stuff I realized it's kinda nice to make it clear we aren't using this info